allennlp/tests/data/tokenizers/pretrained_transformer_tokenizer_test.py

```diff
@@ -50,8 +45,8 @@ def test_splits_cased_bert(self):
             ",",
             "[MASK]",
             "Allen",
-            "##NL",
-            "##P",
+            "NL",
+            "P",
```

Comment:
Why these were wrong?

Reply:
The new code sets `token.text` as a slice from the original text, using the offsets provided by the new tokenizers. The original test will never contain the `##` characters that we had there before.